bin/export-record-barcodes

```diff
@@ -29,14 +29,32 @@ REDCAP_API_TOKEN_SCAN = lambda lang, purview: \
     environ["REDCAP_API_TOKEN_%s" % lang.upper().replace("-", "_")] if purview == "ph" else \
     environ["REDCAP_API_TOKEN_%s_%s" % (purview.upper().replace("-", "_"), lang.upper().replace("-", "_"))]
 REDCAP_API_TOKEN_UW_REOPENING = environ["REDCAP_API_TOKEN_UW_REOPENING"]
+REDCAP_API_TOKEN_CHILDCARE = environ["REDCAP_API_TOKEN_CHILDCARE"]
 
 BARCODE_FIELDS = [
     "pre_scan_barcode",
+    "utm_tube_barcode",
     "utm_tube_barcode_2",
     "reenter_barcode",
     "return_utm_barcode",
     "collect_barcode_kiosk",
     "barcode_swabsend",
+    "barcode_1",
+    "barcode_2",
+    "barcode_3",
+    "barcode_4",
+    "barcode_5",
+    "barcode_6",
+    "barcode_7",
+    "barcode_8",
+    "barcode_9",
+    "barcode_10",
+    "barcode_11",
+    "barcode_12",
+    "barcode_13",
+    "barcode_14",
+    "barcode_15",
+    "barcode_16",
```

Comment:
Definitely not a blocker but it might be nice to generate these and store them in a variable doing something like 
```
sch_barcodes = [ f'barcode_{i + 1}' for i in range(16) ]
```

Reply:
Sure thing, pushed up in a new commit. 
I wanted to dry up the yaml file too but wasn't successful - my idea to use anchors to interpolate reused values in the multiline sql statement string wasn't working well (I mention this in case someone else has an idea for how to improve it!)

Reply:
Interesting. Thanks for trying it out. I don't have any ideas off hand on how to fix that.